manifests/crd.yaml

```diff
@@ -1604,4 +1604,4 @@ spec:
     served: true
     storage: true
     subresources:
-      status: {}
+      status: {}
```

Comment:
I guess you removed the trailing newline in both these files. Please add the trialing newline.